crates/core/tedge/src/cli/refresh_bridges.rs

```diff
@@ -0,0 +1,91 @@
+use camino::Utf8PathBuf;
+use tedge_config::TEdgeConfig;
+use tedge_config::TEdgeConfigLocation;
+
+use super::common::Cloud;
+use super::connect::ConnectError;
+use crate::bridge::BridgeConfig;
+use crate::bridge::CommonMosquittoConfig;
+use crate::bridge::TEDGE_BRIDGE_CONF_DIR_PATH;
+use crate::command::BuildContext;
+use crate::command::Command;
+
+pub struct RefreshBridgesCmd {
+    config: TEdgeConfig,
+    config_location: TEdgeConfigLocation,
+}
+
+impl Command for RefreshBridgesCmd {
+    fn description(&self) -> String {
+        "Refresh all currently active mosquitto bridges".to_string()
+    }
+
+    fn execute(&self) -> anyhow::Result<()> {
+        let clouds = established_bridges(&self.config_location);
+
+        if clouds.is_empty() {
+            println!("No bridges to refresh.");
+        }
+
+        let config_location = &self.config_location;
+        let common_mosquitto_config = super::connect::mosquitto_config(&self.config);
+
+        for cloud in clouds {
+            println!("Refreshing bridge {cloud}");
+
+            let bridge_config = super::connect::bridge_config(&self.config, cloud)?;
+            refresh_bridge(&bridge_config, &common_mosquitto_config, config_location)?;
+        }
+
+        Ok(())
```

Comment:
What can be useful is to detect that at least one bridge has been updated and that mosquitto must be restarted.

I would tell the user that mosquitto must be restarted.

Reply:
> I would tell the user that mosquitto must be restarted.

I'd rather restart mosquitto automatically as part of the command. When upgrading, users generally expect that sometimes services will be restarted, right?

Reply:
Added automatic mosquitto restart.